docs/getting-started.md

```diff
@@ -192,7 +188,10 @@ To do this we will simply call `Kill` on the task after waiting a couple of seco
 	}
 
 	status := <-exitStatusC
-	fmt.Printf("redis-server exited with status: %d\n", status)
+	if status.Err != nil {
```

Comment:
I like the suggestion. It's a bit more natural to check the error return from a function rather than from a struct.